packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.tsx

```diff
@@ -69,6 +71,11 @@ export function ComboboxContentList(props: ComboboxListProps): JSX.Element {
                 />
               );
             })}
+          {props.loading && (
+            <div className={styles.loadingContainer}>
+              <Glimmer size="larger" />
```

Comment:
not sure how many of these to display, we won't know how many results to expect until it's done and then you don't need a glimmer

we could show a glimmer for each of the options we had before you started searching? we might get back more or less after the search though, tough to say!

Reply:
> a glimmer for each of the options we had before you started searching

This is my preference if we can. Playing with it right now, the wrapper collapsing when it has multiple options feels a bit shaky.

Also, how does it look if we use `Glimmer.Text` here? (just trying to approximate the elements we're replacing as closely as possible)

Reply:
![Kapture 2024-01-17 at 13 23 13](https://github.com/GetJobber/atlantis/assets/11843215/03413cb4-5864-4967-b1b7-d3d1ec2a4f6f)

here's what it looks like with the `Glimmer.Text`

hmm ok I can see about having a glimmer for each of the initial options. trying to think through if there's the potential for that going sideways where you have 0 initial options, and then search + load and that could be strange

if I can't get that working, is there a static number you'd be happy with?

Reply:
![image](https://github.com/GetJobber/atlantis/assets/11843215/377309f4-13ab-4ca8-b843-8e1289f74d05)

Reply:
Nice, thank you 🙏